addon/templates/components/dashboard-week.hbs

```diff
@@ -1,16 +1,13 @@
 <div class="weeklylink" data-test-weekly-link>
   {{t "general.view"}}:
-  {{#link-to
-    "weeklyevents"
-    (query-params expanded=expanded)
-  }}
-    {{t "general.lastNextAllWeeks"}}
```

Comment:
Can you also remove `general.lastNextAllWeeks` from the translation files, I don't think we're using it anywhere.

Reply:
Sure. Dave and I thought it might be useful to have it for future reference.

Reply:
makes sense, but we can add it back later. If we don't take it out now it will stick around forever.